sagemaker-featurestore/feature_store_introduction.ipynb

```diff
@@ -473,6 +526,62 @@
     "all_records"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "If we want to load data for the added features into the updated customers_feature_group, use `PutRecord` operation."
+   ]
+  },
```

Comment:
What do you think about moving the schema update code down here right before ingesting records with the new columns? It doesn't seem very useful to create a feature group, immediately update its schema, break up ingestion in to 2 consecutive parts (old schema and new schema).

Reply:
agree. revision made in the new version.